### ISSUE TITLE
Articles: Show party division in news details

### ISSUE THREAD

**Issue by iantrieschmann** [verdigado/gruene-app#214]
**The details view of each article displays the entire contents of the respective article.**

~~Author~~Division, headline, date and article text can be found beneath the article photo - a classic readable article page. The article text can contain clickable links, the attachments (#559) and displays common text editor formatting.

Since most things were already done in #211, the only thing missing here is displaying the division (see https://github.com/verdigado/gruene-app/issues/214#issuecomment-2671002988).

~~Blocked by: https://github.com/verdigado/gruene-api/issues/145~~

![grafik](https://github.com/user-attachments/assets/88968114-1d1f-4651-a41e-9106b3fa861e)

Prototype: https://www.figma.com/proto/7wE0Iveas3GGb8u4oL0714/App-Design-Mitglieder-App-(1)-(Copy)?node-id=2595-11155&t=b8swD5IOAFfZ0OSo-1&starting-point-node-id=2425%3A17429

**Comment by JuliaFreitagBGSt** [comment]
@maxpfe pls link new Issue here regarding Article in Aktuelles

**Comment by JuliaFreitagBGSt** [comment]
@leagerndt pls provide any new screenshots if necessary and check user-flow

**Comment by JuliaFreitagBGSt** [comment]
Rfd @maxpfe

**Comment by steffenkleinle** [comment]
@JuliaFreitagBGSt what is missing here? Isn't this already done (in #211)?

**Comment by JuliaFreitagBGSt** [comment]
At the moment, we dont have the author, the attachments, the article fotos :)

**Comment by steffenkleinle** [comment]
Attachments will be done in #559

**Comment by JuliaFreitagBGSt** [comment]
@steffenkleinle We have received feedback that images are not displayed in articles. See example here: https://netz.gruene.de/de/aktuelles/2025-02/zusammen-wachsen-unser-regierungsprogramm-fuer-die-bundestagswahl

The qr code in image format is not displayed in the app. Will this also be fixed by your further developments? I just need an info about that

**Comment by steffenkleinle** [comment]
Article fotos already done in https://github.com/verdigado/gruene-app/pull/590. Therefore only author is missing.

**Comment by iantrieschmann** [comment]
The article author(s) in terms of the specific person(s) who published the article, no. However, like it is on netz.gruene.de (see attached screenshot), it would be useful to show the party division [Gliederung] responsible for the article _as the author_ so that the reader knows while in the article from which division the article stems. This information is missing while the article is open in-App.

Is this possible, @steffenkleinle?

@JuliaFreitagBGSt CMIIW

![Image](https://github.com/user-attachments/assets/15c89262-4dd3-4638-9ed9-2b02e1edcdac)
